components/search/SearchInterface.vue

```diff
@@ -322,6 +322,12 @@
         this.$store.commit('search/setView', this.$route.query.view);
       }
     },
+
+    mounted() {
+      this.$root.$on('bv::dropdown::hidden', (dropdown) => {
```

Comment:
Instead of this, can't we use `@hidden` on the `b-dropdown` template element in FormDropdown.vue?

Reply:
We can Indeed, and that's updated.